examples/document_creation/browser/package.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I'd like for the examples to be as lightweight as possible, and using `browserify`, `watchify`, and `babel` seems pretty noisy. Many users of `pdf-lib` will not be familiar with these tools, and having them in the examples adds significant extra overhead for these users.

What do you think about just having an `index.html` and `index.js` file for this example? (No `npm` or `package.json`). We could use the [UMD builds from unpkg](https://github.com/Hopding/pdf-lib#umd-module) via a `<script>` tag. This should remove the need for `browserify`. And as for Babel, I don't think we need it at all. As far as I'm aware, all modern browsers support the ES6 syntax (async/await, destructuring, etc...) used in the `index.js` file.

Reply:
Sure, I'll take a stab at it. Seems like module support is equally on par with `async/await`, so I could put the user scripts in a single file.

Btw, should I go for the shared main module approach (the latter) or separate folders?

Reply:
I think I’d favor keeping them split, since that makes each example a bit more straightforward.

But I’m also open to trying to share just the assets. If you want to try that, I’m curious what it’d look like with a simplified browser version of the code.